stable/mlrun/values.yaml

```diff
@@ -447,8 +446,11 @@ ui:
 
   priorityClassName: ""
 
-  podSecurityContext: {}
-  # fsGroup: 2000
+  podSecurityContext:
```

Comment:
leave empty, let user decide (or do you have to have that set up for nginx unprivileged? if not, leave empty

Reply:
removed